solutions/search/esql-for-search.md

```diff
@@ -2,8 +2,8 @@
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/8.18/esql-for-search.html
 applies_to:
-  stack: preview
-  serverless: preview
+  stack: ga 9.1.0
+  serverless: ga 9.1.0
```

Comment:
```suggestion
  stack: preview 9.0, ga 9.1
  serverless: ga
```

Sorry I confused myself here with the new metadata, this is the correct syntax `:homer-disappearing:`

Reply:
Looks good, I was actually going to ask about that 😅